front/src/config/i18n/fr.json

```diff
@@ -1953,7 +1953,7 @@
     "button": "Récupérez votre mot de passe",
     "wrongEmailError": "Ce n'est pas un e-mail valide",
     "networkError": "Erreur réseau : nous n'avons pas réussi à contacter votre instance Gladys. Êtes-vous connecté au réseau ? Votre instance Gladys est-elle en cours d'exécution ?",
-    "unknownError": "Une erreur inconnue s'est produite. Veuillez réessayer ou poster un message sur Gladys community.",
+    "unknownError": "Une erreur inconnue s'est produite. Veuillez réessayer ou nous contacter sur la <a href=\"https://community.gladysassistant.com\">communauté Gladys</a>.",
```

Comment:
It'll not work if the component is not modified.

Also, don't forget  target=\"_blank\" rel=\"noopener noreferrer\"